functions/channelCapture/channelCaptureHandlers.private.ts

```diff
@@ -360,7 +360,13 @@ export const handleChannelCapture = async (
   ]);
 
   const { ENVIRONMENT, HELPLINE_CODE } = context;
-  const languageSanitized = language.replace('-', '_'); // Lex doesn't accept '-'
+  let languageSanitized = language.replace('-', '_'); // Lex doesn't accept '-'
+
+  // This is used to match all digits (0-9) and replace them with no space since Lex doesn't accept numbers
+  if (/\d/.test(languageSanitized)) {
```

Comment:
Do we need an if wrapper? Will replace return the same string if the regex isn't found?

Reply:
@robert-bo-davis  - If there is no number the if block is ignored. So the initial `languageSanitized` will return its original value.